typescript/lambda-cron/index.ts

```diff
@@ -12,14 +12,13 @@ export class LambdaCronStack extends cdk.Stack {
     const lambdaFn = new lambda.Function(this, 'Singleton', {
       code: new lambda.InlineCode(fs.readFileSync('lambda-handler.py', { encoding: 'utf-8' })),
       handler: 'index.main',
-      timeout: 300,
```

Comment:
Replace with updated timeout - timeout=core.Duration.seconds(300) (core = aws-cdk.core) in python - don't know what it is in TS.

Reply:
@tom98784 is correct.
``` typescript
timeout: cdk.Duration.seconds(300),
```
is appropriate here.

Reply:
See my comment above.

Reply:
3c14df84b432527c31e4e4a05a6fcba73e59d0ca

Reply:
@tbadlov this one is not yet resolved. I'm ready to accept the rest, but for consistency-sake, I would like to maintain the same generated resources from the previous version of the project, so the 300 second duration is important.

Reply:
done. 9d935379bee42767781eb4b03b2fb28502e4f678